typescript/infra/config/environments/testnet4/agent.ts

```diff
@@ -225,6 +228,20 @@ const scraperResources = {
   },
 };
 
+const defaultIsmCacheConfig: IsmCacheConfig = {
+  // Default ISM Routing ISMs change configs based off message content,
+  // so they are not specified here.
+  moduleTypes: [
+    ModuleType.AGGREGATION,
+    ModuleType.MERKLE_ROOT_MULTISIG,
+    ModuleType.MESSAGE_ID_MULTISIG,
+  ],
+  // SVM is explicitly not cached as the default ISM is a multisig ISM
+  // that routes internally.
+  chains: ethereumChainNames,
+  cachePolicy: IsmCachePolicy.IsmSpecific,
+};
```

Comment:
do we anticipate splitting the strategies for mainnet3/testnet4? if not could we move this config somewhere more central

Reply:
I think for now the flexibility will be nice just in case there are issues in a pinch around the chains in here